### ISSUE TITLE
Add twitter button to main page.

### ISSUE THREAD

**Issue by dpsnowden** [ioos/catalog#343]
@usioosgov

**Comment by lukecampbell** [comment]
Did we want a button to compliment what we have?
Like maybe a social networking bar that has both twitter and facebook in one small bar?

![screen shot 2015-01-22 at 3 30 16 pm](https://cloud.githubusercontent.com/assets/1155163/5863952/a69e5096-a24b-11e4-93c8-3cfd701b31c0.png)

**Comment by kknee** [comment]
Yes, we wanted twitter next to or below FB.  A small bar would be nice.

![image](https://cloud.githubusercontent.com/assets/196868/5863974/deea6cfa-a24b-11e4-9ba4-d2a9d76ac144.png)
